6.2.2000_run.py

```diff
@@ -0,0 +1,14 @@
+import time
+
+
+def timer(f, *args, **kwargs):
+    st = time.time()
+    n = list(f(*args, **kwargs))
+    et = time.time()
+    elapsed_time = et - st
+    return elapsed_time
```

Comment:
rename 'st' 'n' f' and 'et' Variable names - https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0 - variable names, abbreviation reference is missing

Reply:
fixed it